package.json

```diff
@@ -13,9 +13,12 @@
     "@fortawesome/fontawesome-free": "^6.5.1",
     "axios": "^1.6.7",
     "bootstrap": "^5.3.3",
+    "jwt-decode": "^4.0.0",
```

Comment:
I think we can use this API instead of using jwt-decode, as the decoded data doesn't contain the image, and we will need the image for the navbar.
So I suggest removing the jwt-decode and after the user login, we make another API call to this API to get all user data and store them for use.
![image](https://github.com/Elias-lap/ProjectMangement2/assets/64327685/abc93d4e-01b8-4b9c-bceb-f459d367c00b)